catalog/core/catalog-core-urlresourcereader/src/test/java/ddf/catalog/resource/impl/URLResourceReaderTest.java

```diff
@@ -366,12 +366,7 @@ public void testUrlToNonExistentFile() throws Exception {
   public void testHTTPReturnsFileNameWithoutPath() throws Exception {
     URI uri = new URI(HTTP_SCHEME_PLUS_SEP + HOST + TEST_PATH + JPEG_FILE_NAME_1);
 
-    verifyFileFromURLResourceReader(uri, JPEG_FILE_NAME_1, JPEG_MIME_TYPE);
-
-    Path pathForUri = Paths.get(ABSOLUTE_PATH, TEST_PATH, JPEG_FILE_NAME_1);
-    uri = pathForUri.toUri();
-
-    verifyFileFromURLResourceReader(uri, JPEG_FILE_NAME_1, JPEG_MIME_TYPE);
```

Comment:
I think that: https://github.com/codice/ddf/blob/3efc289a995afbddf1170c716233a34ee82eaab8/catalog/core/catalog-core-urlresourcereader/src/test/java/ddf/catalog/resource/impl/ResourceReaderTest.java#L371-L374
is duplicated in 
https://github.com/codice/ddf/blob/3efc289a995afbddf1170c716233a34ee82eaab8/catalog/core/catalog-core-urlresourcereader/src/test/java/ddf/catalog/resource/impl/ResourceReaderTest.java#L588-L592
and can be removed.